include/library/spdm_lib_config.h

```diff
@@ -166,6 +166,11 @@
 #define LIBSPDM_SM3_256_SUPPORT 1
 #endif
 
+/* This can be set to 0 for the device which does not need X509 parser.*/
+#ifndef LIBSPDM_X509_SUPPORT
```

Comment:
I would call this `LIBSPDM_CERT_PARSE_SUPPORT` since there may come a time where we support non-x509 certificates.

Reply:
done. Use LIBSPDM_CERT_PARSE_SUPPORT.